gui/src/components/Taipy/Slider.spec.tsx

```diff
@@ -107,6 +114,13 @@ describe("Slider Component", () => {
         expect(elts).toHaveLength(1);
         expect(elts[0].tagName).toBe("P");
     });
+    it("holds a lov range", async () => {
+        const { getByDisplayValue } = render(<Slider value={["B", "C"]} defaultLov={'[["A", "A"], ["B", "B"], ["C", "C"], ["D", "D"]]'} />);
```

Comment:
idem

Reply:
idem

Reply:
Fixed.

Reply:
I disagree. The value is not a string, even from the backend standpoint

Reply:
indeed, only for defaultValue